### ISSUE TITLE
1.19.3 PlayerInfoUpdatePacket not sending listed with just ADD_PLAYER action

### ISSUE THREAD

**Issue by fullwall** [SpigotMC/BungeeCord#3412]
### Bungeecord version

latest

### Client version

1.19.3

### Bungeecord plugins

None

### The bug

I am setting ```listed``` to false with the PlayerInfoUpdatePacket ADD_PLAYER action however BungeeCord appears to only proxy this boolean if the action also includes UPDATE_LISTED. I can confirm the client will set the tablisted status accordingly even if set in an ADD_PLAYER action and this behaviour appears to be in the unofficial protocol wiki https://wiki.vg/Protocol#Player_Info

### Checking

- [X] I am using BungeeCord and **not a fork**. Issues with forks should not be reported here.
- [X] I think this is **not** an issue with a bungeecord plugin.
- [ ] I have not read these checkboxes and therefore I just ticked them all.
- [X] This is not a question or plugin creation help request.

**Comment by md-5** [comment]
You've linked 1.19.2 protocol documentation (which doesn't even have the listed field).

The Vanilla code also disagrees with you, the boolean is only read in the UPDATE_LISTED action, not the ADD_PLAYER action. If it was read in both and bungee didn't, then you would be disconnected with a protocol error. 

1.19.3 dramatically overhauled this packet such that you can have multiple actions in the same packet to add more fields.

**Comment by md-5** [comment]
Not sure where the previous comment went, but listed = true:

![image](https://user-images.githubusercontent.com/1007849/208268809-4f6bb647-0002-4a6f-86d9-21c9262fde6a.png)

And listed=false (no visible tab list):

![image](https://user-images.githubusercontent.com/1007849/208268813-133102c2-4442-4d8e-9e23-e2366c7a296a.png)

Seems to be working fine for me.

**Comment by mcmonkey4eva** [comment]
That last comment was deleted after we did some testing of our own and couldn't replicate it... then started noticing other oddities with the multiple users all reporting the issue.

After some investigating we discovered the users reporting the issue in relation to their bungee netowrk, weren't just on bungee, they also forgot to mention they were connecting through ViaVersion on a 1.19(.0) client.

So, yeah, this is definitely not a Bungee issue.
